### DISCUSSION TITLE
How to display minicart items in left sidebar?

### DISCUSSION THREAD

**Discussion by webcreative24** [breezefront/community#207] [1. Help]
Hello. How I can show mini cart content (items) in a separate div? I need to show mini cart items on the right side([example](https://monosnap.com/direct/jNvsCh53MNK2dktcdpBgO7UvRuSBmO))

thanks @vovayatsyuk

**Comment by vovayatsyuk** [comment]
It's not currently possible to easily render header minicart in other places due to hardcoded logic (it must be inside dropdown).

**Reply by webcreative24** [reply]
Also not possible to re-use some components to show cart items in some other place?

**Reply by vovayatsyuk** [reply]
It's possible, but not easy to achieve. Here is starter point for you:

**your_theme/Magento_Checkout/layout/breeze_default.xml**

```xml
<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <body>
        <referenceBlock name="breeze.js">
            <arguments>
                <argument name="bundles" xsi:type="array">
                    <item name="default" xsi:type="array">
                        <item name="items" xsi:type="array">
                            <item name="minicart-expanded-mixin" xsi:type="string">js/breeze/minicart-expanded-mixin</item>
                        </item>
                    </item>
                </argument>
            </arguments>
        </referenceBlock>

        <referenceBlock name="content">
            <block class="Magento\Checkout\Block\Cart\Sidebar" before="-" name="minicart_expanded" template="Magento_Checkout::cart/minicart_expanded.phtml"/>
        </referenceBlock>
    </body>
</page>
```

**your_theme/Magento_Checkout/templates/cart/minicart_expanded.phtml**

```js
<div data-block="minicart-expanded" class="minicart-wrapper">
    <div class="block block-minicart">
        <div data-bind="scope: 'minicart_content'"><!-- ko template: getTemplate() --><!-- /ko --></div>
    </div>

    <script type="text/x-magento-init">
    {
        "[data-block='minicart-expanded']": {
            "Magento_Ui/js/core/app": <?= $block->getJsLayout() ?>
        }
    }
    </script>
</div>
```

**your_theme/web/js/breeze/minicart-expanded-mixin.js**

```js
(() => {
    'use strict';

    $.mixin('minicart', {
        create: function (original) {
            if (this.element.closest('[data-block="minicart-expanded"]').length) {
                this.minicartSelector = '[data-block="minicart-expanded"]';
                this.minicart().sidebar(this.getSidebarSettings());
            }

            original();
        }
    });
})();
```

Result: 

![image](https://github.com/breezefront/community/assets/306080/63898cd9-c49a-4045-82c1-3260fda58323)

**Comment by webcreative24** [comment]
thank you

**Reply by webcreative24** [reply]
@vovayatsyuk Hi
I tried to override minicart-content html template but it's not working with Breeze...
I copied the html template into custom theme app/design/frontend/Vendor/theme/Magento_Checkout/web/template/minicart/content.html but it still uses the magento core template. is there something specific with a breeze theme?
Thanks

**Reply by webcreative24** [reply]
@vovayatsyuk found it in vendor/swissup/module-breeze/view/frontend/templates/ko/checkout/minicart.phtml
